readthedocs/templates/core/project_details.html

```diff
@@ -41,6 +41,15 @@ <h3>{% trans "Versions" %}</h3>
           {% if request.user|is_admin:project %}
             <ul class="module-item-menu">
               <li><a href="{% url "project_version_detail" project.slug version.slug %}">{% trans "Edit" %}</a></li>
+              <li>
+                <span class="right quiet" style="margin:10px 5px 0px 0px;">
+                  {% if version.built %}
```

Comment:
This isn't checking if the build is passing, but only that it has ever passed. We should properly check the last build for this version to get this data.

Reply:
@ericholscher 
I have implemented a custom template tag for this purpose.